tests/ui/traits/vtable/multiple-markers.stderr

```diff
@@ -16,9 +13,7 @@ error: vtable entries for `<S as B>`: [
            MetadataDropInPlace,
            MetadataSize,
            MetadataAlign,
-           TraitVPtr(<S as M1>),
            Method(<S as T>::method),
-           TraitVPtr(<S as T>),
            TraitVPtr(<S as M2>),
```

Comment:
Why is TraitVPtr needed for M2?

Reply:
The short answer: it is not needed. The longer answer is that we currently keep the order of all things from the traits, so since `M2` comes after `T`, it can't be inlined _there_. I've filled https://github.com/rust-lang/rust/issues/114942 to track this.